library/packages/src/modules/PackageCallbacks.rb

```diff
@@ -416,7 +417,8 @@ def ProgressPackage(percent)
     #   a blank string is returned (so no decision is made).
     def pkg_gpg_check(data)
       log.debug("pkgGpgCheck data: #{data}")
-      Linuxrc.InstallInf("Insecure") == "1" ? "I" : ""
+      return "I" if Stage.initial && Linuxrc.InstallInf("Insecure") == "1"
```

Comment:
Um, IMHO that's difficult to read... what about something like

```ruby
insecure = Stage.initial && Linuxrc.InstallInf("Insecure") == "1"
insecure ? "I" : ""
```
???

BTW Rubocop is OK with that :wink: